app/main.py

```diff
@@ -1 +1,12 @@
-# write your code here
+file_name = input("Please enter the name of the file: ")
+file_content = ""
+
+while True:
+    new_line = input("Waiting for input: ")
+    if new_line == "stop":
+        break
+    else:
+        file_content += new_line + "\n"
+
+with open(file_name, "w") as f:
```

Comment:
In task says: When the app is launched, ask the user to enter the file name.
Ex: 
`Enter name of the file: name1`
Not -  `Enter name of the file: name1.txt`
please correct: `with open(file_name, "w") as f:` -> `with open(file_name + '.txt', 'w') as file:`